backend/settings/base.py

```diff
@@ -58,6 +61,11 @@ class Base(Configuration):
         "django.middleware.clickjacking.XFrameOptionsMiddleware",
     ]
 
+    CORS_ALLOWED_ORIGINS = [
```

Comment:
Let's keep the defaults empty and set values for each env in corresponding files, e.g. CORS_ALLOWED_ORIGINS = ("http://localhost:3000", "http://127.0.0.1:3000") for `local.py`